devito/propagator.py

```diff
@@ -11,6 +11,8 @@ def __init__(self, name, nt, shape, spc_border=0, forward=True, time_order=0):
         self.t = symbols("t")
         # We assume the dimensions are passed to us in the following order
         # var_order
+        self.cache_blocking = True
+        self.block_sizes = [10]*num_spac_dim
```

Comment:
Where does the default block size come from? How would a user change the block size?